src/client/enterprise/enterprise.proto

```diff
@@ -51,6 +51,17 @@ enum State {
 message GetStateResponse {
   State state = 1;
   TokenInfo info = 2;
+
+  // activation_code will always be an empty string,
+  // call GetEnterpriseCode to get the activation code
+  string activation_code = 3;
```

Comment:
You could also remove the field by deleting it and adding `reserved: 3;` to the top of the message. I'm not sure if we should avoid doing that in the 1.12 branch for semver (so that client code that reads the field continues to compile? Though if it's no longer being set then that client code would still break, so leaving it actually seems like introducing a silent failure, so maybe we should remove it? I'm honestly not sure...). But either way I think it could make sense to remove and reserve the field in `master`, and I think doing so could allow you to catch any internal calls that read it.

Reply:
This is a good point - I didn't think the enterprise code was used anywhere, but we're using it for our Segment metrics 🤦‍♀️ 

I'll talk to the marketing team about how we want to handle this, maybe we can hash the token to get a unique identifier that random people from the internet can't grab for themselves.

Reply:
OK, I've updated this after talking to Dan - now `GetState` strips the signature but leaves in all the fields Marketing uses. In 2.0 I would like to move GetState to an internal service and/or revisit how we identify clusters but I think it's worth having this fix on master and 1.12.x as a stopgap.